internal/services/containerapps/container_app_resource.go

```diff
@@ -296,7 +296,7 @@ func (r ContainerAppResource) Read() sdk.ResourceFunc {
 
 func (r ContainerAppResource) Delete() sdk.ResourceFunc {
 	return sdk.ResourceFunc{
-		Timeout: 30 * time.Minute,
+		Timeout: 60 * time.Minute,
```

Comment:
Why change the timeout? If needed, you would also need to update the documentation:
> `delete` - (Defaults to 30 minutes) Used when deleting the Container App.

Reply:
This was a change made to facilitate debugging during testing and mistakenly included.